storage/ios/Plugin/Storage.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+public struct StorageConfiguration {
+    var group = "CapacitorStorage"
+}
```

Comment:
If we're willing to drop Obj-C interoperability for this class, it would be cleaner and more idiomatic to change the group into an enum. That would shift the burden of checking for the magic "NativeStorage" string to the plugin class which is closer to where it belongs.

Also note the change to `let`, which should be preferred in this case because the group shouldn't be changed after initialization.
```suggestion
public struct StorageConfiguration {
    enum Group {
        case native, prefixed(String)
    }
    let group: Group
    
    init(for group: Group = .prefixed("CapacitorStorage")) {
        self.group = group
    }
}
```